src/test/pmem_deep_persist/grep0.log.match

```diff
@@ -1,2 +1,2 @@
 1
-0
+$(OPT)0
```

Comment:
Please echo 0 to grep0.log on WIndows with an explanation instead of making this line optional.